kong-ngx-build

```diff
@@ -197,7 +201,7 @@ main() {
       notice "Patched OpenResty found, removing..."
       rm -rf $OPENRESTY_DOWNLOAD
 
-    elif [[ $OPENRESTY_PATCHES == 1 && ! -f $OPENRESTY_DOWNLOAD/bundle/.patch_applied ]]; then
+    elif [[ $OPENRESTY_PATCHES != 0 && ! -f $OPENRESTY_DOWNLOAD/bundle/.patch_applied ]]; then
```

Comment:
Hopefully it is fine to use the same variable to signify "no patches" and "patches from this branch" (different types), but happy to add another variable if necessary. I don't think users (mostly ourselves) should ever try to specify strange branch names in this option anyway.